library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -12,6 +13,15 @@
 
     .rubric-item {
         position: relative;
+
+        .divDescription{
```

Comment:
could you stick to using hypens, rather than camelcase?

Reply:
Thanks for the review @adrianfish, I've marked this PR as work in progress some hours ago because we expect more work here, I'm not convinced with the implementation.